app/api/settings/specs/routes.spec.js

```diff
@@ -12,6 +12,10 @@ describe('relationtypes routes', () => {
   });
 
   describe('GET', () => {
+    it('should have a validation schema', () => {
+      expect(routes.get.validation('/api/settings')).toMatchSnapshot();
+    });
```

Comment:
@RafaPolit the GET endpoint actual does not have any validation since it does not accept any data. I believe that's why the snapshot matches `undefined`. Maybe the test was put there for consistency. Should the test case be removed?